Pipfile

```diff
@@ -28,6 +28,7 @@ coverage = "==7.2.2"
 pytest-cov = "==4.0.0"
 ipython = "==8.12.0"
 pytest-asyncio = "==0.21.0"
+rpy2 = "==3.5.10"
```

Comment:
move rpy2 temporally to dev-packages so that docker image can be built. create a ticket to fix it later. 
https://kbase-jira.atlassian.net/browse/RE2022-149